devstats/projects.yaml

```diff
@@ -0,0 +1,14 @@
+projects:
+  kubeflow:
+    order: 1
+    command_line:
+      - kubeflow
+    # TODO(jlewi): Move the start date back in time once we are done doing some initial testing
+    start_date: 2018-04-16T00:00:00Z
+    # Can we get rid of join date since we aren't part of CNCF yet?
+    join_date: 2018-04-16T00:00:00Z
```

Comment:
You can set it to `null` just like opencontainers project in CNCF repo:
See: https://github.com/cncf/devstats/blob/master/projects.yaml#L241